src/graph.rs

```diff
@@ -137,44 +137,93 @@ impl Range {
 }
 
 #[derive(Debug, Clone)]
-pub enum ModuleGraphError {
+pub enum ModuleError {
+  LoadingErr(ModuleSpecifier, Option<Range>, Arc<anyhow::Error>),
```

Comment:
I think we should eventually change these all to object syntax so it's more understandable.